com.ibm.streamsx.inet/impl/java/src/com/ibm/streamsx/inet/http/HTTPPostOper.java

```diff
@@ -157,6 +158,11 @@ public void setMaxRetries(int val) {
 	public void setRetryDelay(double val) {
 		this.retryDelay = val;
 	}
+	@Parameter(optional=true, description="Optional parameter specifies amount of time (in seconds) that the operator waits for the connection for to be established.")
```

Comment:
No change to the default timeout - if the value is not set then the default that the underlying class provides stands, which is not stated as far as I can find in it's doc. 

I came to the  60 second timeout by observation.